### PR TITLE
fix username not shown for OIDC users

### LINKED ISSUE #4214: Username is not enriched in users page while using OIDC
Username is not enriched when browsing the users page with OIDC.

image

### PR DESCRIPTION
**Pull request by eladlachmi**
Resolves #4214

### CODE REVIEW

File: webui/src/pages/auth/users/user/policies.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 import {AuthLayout} from "../../../../lib/components/auth/layout";
-import {UserHeader} from "../../../../lib/components/auth/nav";
+import { UserHeaderWithContext } from "./userHeaderWithContext";
```
Comment (nopcoder) [inline webui/src/pages/auth/users/user/policies.jsx:4]: Align the format "{ something }" vs "{something}" - maybe we should have something like prettier

**Review by nopcoder — CHANGES_REQUESTED** [review]
Small code and style comments

File: webui/src/pages/auth/users/user/policies.jsx (hunk 1 above)
Comment (eladlachmi) [reply webui/src/pages/auth/users/user/policies.jsx:4]: Yes, we should and we will :)

**Review by nopcoder — APPROVED** [review]
LGTM!

**Pull request merged**